CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # @rhds/tokens
 
+## 1.0.0-beta.5
```

Comment:
hold this release until @zeroedin can adjust the breakpoints to a mobile-first stance

Reply:
except since he already did, approving :D